docker-wordpress-mysql/azuredeploy-parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "value": "unqiueStorageAccount"
+        },
+	"location": {
+	    "value": "West US"
+	},
+        "mysqlPassword": {
+            "value": "password"
+        },
+        "adminUsername": {
+            "value": "userName"
+        },
+        "adminPassword": {
+            "value": "password"
+        },
+        "dnsNameForPublicIP": {
+            "value": "uniqueDNS"
+        }
```

Comment:
Hmm since I copy pasted this from corey's template, I guess it's a problem there as well. The parameter is rendered in the UI, though.